processor/groupbytraceprocessor/processor_test.go

```diff
@@ -179,15 +179,10 @@ func TestProcessBatchDoesntFail(t *testing.T) {
 	span := ils.Spans().At(0)
 	span.SetTraceID(traceID)
 	span.SetSpanID(pdata.NewSpanID([8]byte{1, 2, 3, 4}))
-	batch.ResourceSpans().Append(pdata.NewResourceSpans())
 
 	p, err := newGroupByTraceProcessor(logger, st, next, config)
 	require.NoError(t, err)
 
-	// sanity check
-	err = p.processResourceSpans(batch.ResourceSpans().At(1))
```

Comment:
Why was this removed? This was a sanity check...

Reply:
ping @bogdandrutu

Reply:
Sorry, definitely missed this, looks like test does not exist anymore